src/main/java/reactor/netty/http/websocket/WebSocketConfigurer.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2011-Present Pivotal Software Inc, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.netty.http.websocket;
+
+import javax.annotation.Nullable;
+import java.util.Objects;
+
+/**
+ * Configurer for websocket
+ *
+ * @author Dmitrii Borin
+ */
+public interface WebSocketConfigurer {
```

Comment:
I think `Configurer` is not the right term. It is not configuring anything by itself. It is simply a holder of the configuration. I don't find any such existing class names in Reactor Netty either. Perhaps `WebSocketConfig` or something like that?

Reply:
@rstoyanchev What about `WebSocketSpec` as we use `Spec` in other places in the code where we specify a configuration?